.github/workflows/codecov.yaml

```diff
@@ -14,13 +14,26 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
-      - name: Setup Sui CLI
-        uses: ./.github/actions/install-sui-cli
+      # This step ensures that the "LITERAL" value is given to the actions.
+      # Using `jq` sends it uninterepted to the action which is sort of "hackish" and doesn't ensure the separation of concerns.
+      - name: Get SUI and NodeJS Versions
+        shell: bash
+        run: |
+          SUI_VERSION=$(jq -r '.SUI_VERSION' "version.json")
+          echo "SUI_VERSION=$SUI_VERSION" >> $GITHUB_ENV
+
+          NODEJS_VERSION=$(jq -r '.engines.node' "package.json")
```

Comment:
node engine value is set to `>=18` and not `18`, is that a valid node version value?

Reply:
But does that mean it could install node 19+ then? We want to install only node 18.x for consistency